src/app/layout.tsx

```diff
@@ -1,12 +1,13 @@
-import React from "react";
 import type { Metadata } from "next";
+import React from "react";
 
 import { LayoutMain } from "@/components/layout/LayoutMain";
 import { QueryProvider } from "@/query/QueryProvider";
 import { StoreProvider } from "@/store/StoreProvider";
 
-import "@stellar/design-system/build/styles.min.css";
+import { NetworkByPasswordProvider } from "@/components/NetworkByPasswordProvider";
 import "@/styles/globals.scss";
+import "@stellar/design-system/build/styles.min.css";
```

Comment:
Let's keep the Design System import before the `global.css` to make sure the global overrides work.